src/regrid.jl

```diff
@@ -1,49 +1,56 @@
 """$(TYPEDSIGNATURES)
-Regrid data on `grid_in` onto `grid_out` conservativly (mean-preserving) using the `regridder` matrix.
-`area_out` is the area of each grid cell in `grid_out` and is used to normalize the result,
+Regrid data on `field_in` onto `field_out` conservativly (mean-preserving) using the `regridder` matrix.
+`area_out` is the area of each grid cell in `field_out` and is used to normalize the result,
 if not provided, will recompute this from `regridder`."""
 Base.@propagate_inbounds function regrid!(
-    grid_out::AbstractVector,
-    grid_in::AbstractVector,
+    field_out::AbstractVector,
+    field_in::AbstractVector,
```

Comment:
Should this be `values` instead?

Reply:
Greg and I had a long discussion about terminology, we settled on `field` for data on a grid where grid is a tessellation of a space into polygons. A field would contain values, but values sounds too generic as it could also mean everything else